greenery/snapshots/snapshot_orders.sql

```diff
@@ -0,0 +1,14 @@
+{% snapshot orders_snapshot %}
+
+{{
+    config(
+        target_schema='dbt_zoe_l',
+        unique_key='order_id',
+        strategy='check',
+        check_cols=['status']
```

Comment:
Consider adding a few other columns that may change to check_cols. Like maybe the estimated shipping time could change without the status changing and your snapshot would miss that.